app/helpers/websites_helper.rb

```diff
@@ -12,13 +13,18 @@ def self.createJSON(website)
       ranks          << rank_date_pair
       dates          << site.created_at.strftime('%Y-%m-%d')
     end
+
     series            = []
-    series           << { :name => url, :data => ranks}
+    series           << { :name => url,  :data => ranks}
+   
     res               = {}
-    res['title']      ={:text       =>'Alexa Rank'}
-    res['xAxis']      ={:categories => dates.uniq,
-                        :title      => {text: 'Date(YYYY-mm-dd)'},
-                        :min        => 0
+    res['title']      ={:text           =>'Alexa Rank'}
+
+    res['subtitle']   ={:text           => descript }
+    
+    res['xAxis']      ={:categories     => dates.uniq,
+                        :title          => {text: 'Date(YYYY-mm-dd)'},
+                        :min            => 0
                         }
```

Comment:
Leave a space on either side of `=`.

https://github.com/bbatsov/ruby-style-guide